packages/frontend/package.json

```diff
@@ -35,12 +35,14 @@
         "config-react-env": "./scripts/config-react-env.js",
         "start:staging": "STAGING=true craco start",
         "start": "craco start",
-        "test:e2e": "yarn prepare-fathom && playwright test"
+        "test:e2e": "yarn prepare-fathom && playwright test",
+        "smoke": "yarn playwright test"
```

Comment:
Why a new `smoke` script if we run the same `playwright test` like the script above?

Reply:
yes I wanted to have separate script just for smoke test and the other one for all tests I'will just edit the existing one to be ` "test:e2e": "yarn prepare-fathom && yarn playwright test"` in order to work